src/gui/properties/propertieswidget.cpp

```diff
@@ -143,7 +143,7 @@ PropertiesWidget::PropertiesWidget(QWidget *parent)
     m_ui->vBoxLayoutPeerPage->addWidget(m_peerList);
     // Tab bar
     m_tabBar = new PropTabBar(nullptr);
-    m_tabBar->setContentsMargins(0, 5, 0, 0);
+    m_tabBar->setContentsMargins(0, 5, 0, 5);
```

Comment:
this and the change in .ui file should have its own commit.

Reply:
Also I like no the border style better, so better fix #11618 in another PR instead.

Reply:
I guess you meant by no margins at all
![image](https://user-images.githubusercontent.com/34717789/80833216-b6587200-8c0b-11ea-9b29-95cfd780f977.png)
I feel it's too cramped with no margins